packages/fxa-content-server/app/scripts/templates/confirm_signup_code.mustache

```diff
@@ -29,3 +29,7 @@
         </form>
     </section>
 </div>
+
+<!-- New L10N Strings
+{{#t}}for your Mozilla account{{/t}}
```

Comment:
Can we use this opportunity to fix this so the full sentence is one string?

Similar to what's going on in force_auth.mustache: `{{#unsafeTranslate}}Enter your password <span class="card-subheader" id="subheader">for your Mozilla account</span>{{/unsafeTranslate}}`

This makes the translation very awkward for some languages that have different word order. Double so since the first half of the sentence was already translated months ago and it might not be so apparent to current translators what's going on here.